dev-portal/public/swagger-ui/dist/css/print.css

```diff
@@ -1,1367 +0,0 @@
-/* Original style from softwaremaniacs.org (c) Ivan Sagalaev <Maniac@SoftwareManiacs.Org> */
```

Comment:
We should try to split commits that remove assets/dependencies/libraries and that make changes (this commit contains the changes to move to swagger ui 3 as well as removing a bunch of swagger ui 2 files).

Reply:
Don't agree. That would mean the step between two single commits is totally broken.

Reply:
That's fine. I'd prefer reviewer ease over 'every commit is releasable'. Releasable commits should be tagged, other commits should be assumed to not be guaranteed to work.

Reply:
:+1: